cdap-app-fabric/src/main/java/co/cask/cdap/gateway/handlers/ProgramLifecycleHttpHandler.java

```diff
@@ -1097,16 +1095,8 @@ public void getServiceInstances(HttpRequest request, HttpResponder responder,
     getServiceInstances(request, responder, namespaceId, appId, serviceId, serviceId);
   }
 
-  /**
-   * Return the number of instances for the given runnable of a service.
-   */
-  @GET
-  @Path("/apps/{app-id}/services/{service-id}/runnables/{runnable-name}/instances")
-  public void getServiceInstances(HttpRequest request, HttpResponder responder,
-                                  @PathParam("namespace-id") String namespaceId,
-                                  @PathParam("app-id") String appId,
-                                  @PathParam("service-id") String serviceId,
-                                  @PathParam("runnable-name") String runnableName) {
+  void getServiceInstances(HttpRequest request, HttpResponder responder,
```

Comment:
Both the methods are used in endpoints in the current class - to support the new endpoint for services.

I didn't think it made sense to refactor/break-down further since it's going to change in 3.0 anyway. Thoughts?